catroid/src/org/catrobat/catroid/formulaeditor/FormulaEditorEditText.java

```diff
@@ -167,12 +170,26 @@ public void enterNewFormula(InternFormulaState internFormulaState) {
 		}
 	}
 
+	private Runnable cursorAnimation = new Runnable() {
+		@Override
+		public void run() {
+			paint.setColor((paint.getColor() == 0x00000000) ? 0xff000000 : 0x00000000);
+			invalidate();
+			postDelayed(cursorAnimation, 500);
+		}
+	};
+
+	public Paint getpaint() {
+		return paint;
+	}
+
```

Comment:
Do not implement public getter methods only for testing purpose. (Use f.e. Reflection)